### PR TITLE
About Us work done

### PR DESCRIPTION
**Pull request by dev04sa**
@Abhinav1341  About us part Work done. I implement all necessary components and animations. also I added the social links of Instagram an linkedin review the changes .

Screenshot 2024-01-24 180019

video link : 
[b4d4273e-3540-4714-b547-fc5686bf8ad2.webm](https://github.com/HackSlashNITP/ByteVerse_Website/assets/129666293/3e716029-cc42-4d16-9973-395fc8498575)

### CODE REVIEW

File: byteverse_24/src/app/components/AboutIamge.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+import React from "react";
+
+const AboutIamge = () => {
+  return (
+    <div>
+      <Image
+        src="/byteverse/aboutus.png"
+        alt="About US Logo"
+        className="dark:invert  "
```
Comment (RitwikSingh28) [inline byteverse_24/src/app/components/AboutIamge.jsx]: @dev04sa 
I don't think we would want to invert the colors when a user is in dark mode. For that will alter the color scheme completely. Do remember that most of the users use their phones in dark mode itself.

File: byteverse_24/src/app/components/HeadImage.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Image from 'next/image'
+import React from 'react'
+
+const HeadImage = () => {
+  return (
+    <div className=' flex justify-center items-center h-[190px] w-[173px] ' >
+         <Image
+              src="/GroupFace.svg"
+              alt="Head Logo"
+              className="dark:invert"
```
Comment (RitwikSingh28) [inline byteverse_24/src/app/components/HeadImage.jsx]: Same here. Please remove the class `dark:invert`

File: byteverse_24/src/app/components/KnifeImage.jsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+  const handleAnimationComplete1 = () => {
+    if (animationState1 === "initial") {
+      setAnimationState1("moveLeft");
+    } else if (animationState1 === "moveLeft") {
+      setAnimationState1("center");
+    } else {
+      setAnimationState1("initial");
+    }
+    console.log("Animation 1 complete. Current state:", animationState1);
```
Comment (RitwikSingh28) [inline byteverse_24/src/app/components/KnifeImage.jsx]: @dev04sa 
Always remember, when pushing your code onto production branch, you must remove any logs to the console.

**Review by RitwikSingh28 — CHANGES_REQUESTED** [review]
@dev04sa 
Please look into the comments I have provided, and try to address them by pushing subsequent commits to this PR.

Moreover, there is a glitch I came across. When in mobile view, the `ninja-icon` covers the GDSC Logo. Please try to get that fixed.

**Comment by dev04sa** [comment]
@RitwikSingh28  Sir I have done all the changes as u told please review the changes.
[3af7913c-9302-40c6-904e-a40add0d443f.webm](https://github.com/HackSlashNITP/ByteVerse_Website/assets/129666293/b28d646d-650f-4c9d-bb9a-bd4ecf1cb3d8)

**Comment by RitwikSingh28** [comment]
@dev04sa 
The glitch is still there, if you diminish the viewport further. 
Moreover, in the video you have provided, you may also notice that the social media logos for HackSlash vanish in the mobile view.

**Comment by dev04sa** [comment]
@RitwikSingh28  Sir issue resolved

Screenshot 2024-01-24 230703
[551bb768-b06c-4656-9854-4ea38607bcbc.webm](https://github.com/HackSlashNITP/ByteVerse_Website/assets/129666293/170425b6-9ecd-45b3-b40e-216302c344f7)

**Comment by RitwikSingh28** [comment]
@dev04sa 
Great work! All the issues have been addressed...

**Pull request merged**